jsk_arc2017_common/node_scripts/set_location_in_rosparam.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
Could you please show me how to use this? for location:=boxes. I don't have no params in that case.